src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/integration/LicenceActivationIntegrationTest.kt

```diff
@@ -33,6 +45,18 @@ class LicenceActivationIntegrationTest : IntegrationTestBase() {
       .exchange()
       .expectStatus().isOk
 
+    verify(eventsPublisher, times(5)).publishDomainEvent(eventCaptor.capture(), any())
+
+    assertThat(eventCaptor.allValues[0].eventType).isEqualTo(DomainEventsService.LicenceDomainEventType.LICENCE_ACTIVATED.value)
+
+    assertThat(eventCaptor.allValues[1].eventType).isEqualTo(DomainEventsService.LicenceDomainEventType.LICENCE_ACTIVATED.value)
+
+    assertThat(eventCaptor.allValues[2].eventType).isEqualTo(DomainEventsService.LicenceDomainEventType.LICENCE_ACTIVATED.value)
+
+    assertThat(eventCaptor.allValues[3].eventType).isEqualTo(DomainEventsService.LicenceDomainEventType.LICENCE_INACTIVATED.value)
+
+    assertThat(eventCaptor.allValues[4].eventType).isEqualTo(DomainEventsService.LicenceDomainEventType.LICENCE_INACTIVATED.value)
```

Comment:
Could maybe  
```kt
val activationEvents = eventCaptor.allValues.filter { it.eventType ==  LICENCE_ACTIVATED }
assertThat(activationEvents).hasSize(3)

val inactivatedEvents = eventCaptor.allValues.filter { it.eventType ==  LICENCE_INACTIVATED }
assertThat(inactivatedEvents).hasSize(3)
```

or if you think event order matters, map to event status and to 
```kt
assertThat(eventTypeStatus).containsExactly(LICENCE_ACTIVATED, LICENCE_ACTIVATED, LICENCE_ACTIVATED, LICENCE_INACTIVATED, LICENCE_INACTIVATED)
```